electrum/plugins/bitbox02/bitbox02.py

```diff
@@ -510,6 +510,12 @@ def sign_transaction(
                 )
 
         keypath_account = full_path[:-2]
+
+        format_unit = bitbox02.btc.BTCSignInitRequest.FormatUnit.DEFAULT
+        # Base unit is configured to be "sat":
+        if self.config.get_decimal_point() == 0:
```

Comment:
I avoided `get_base_unit()` here it because I wasn't sure if it could ever change from 'sat' to e.g. 'sats' or 'SAT', breaking this silently.  `get_decimal_point()` looks more robust to me.

Reply:
Ok, makes sense.